flow/client_server/static/internal/groups/create/js/script.js

```diff
@@ -0,0 +1,36 @@
+function onceDocumentReady() {
+	getURLs(startSubmitButtonListener);
```

Comment:
It's a function defined here https://github.com/baxter-cs/roastme/blob/71d5346b08a079db4b9e90a318f6b228c4837c22/flow/client_server/static/internal/util/js/get_auth_server_address.js